libraries/Microsoft.Bot.Builder.Dialogs.Declarative/Converters/InterfaceConverter.cs

```diff
@@ -67,10 +70,20 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
 
             using (new SourceScope(sourceContext, range))
             {
+                string refDialogName = null;
                 if (this.resourceExplorer.IsRef(jToken))
                 {
-                    // We can't do this asynchronously as the Json.NET interface is synchronous
-                    jToken = this.resourceExplorer.ResolveRefAsync(jToken, sourceContext).GetAwaiter().GetResult();
+                    refDialogName = jToken.Value<string>();
```

Comment:
Maybe should use "GetRefTarget()" ? But it seems a private method.

Reply:
ResolveRefAsync is the public interface for resolving reference. I didn't add this, just removed it to the codnition when not cached.